Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,45 @@
 
 ## Exercise I
 
-a)
+a) O(n ^ 3)
 
 
-b)
+b) O(n * (n - 1)/2)
 
 
-c)
+c) O(n)
 
 ## Exercise II
 
-
+arr = array of all floors indication break of not e.g. [False, False, False, True]
+n = number of floors
+c = number of drops + number of eggs broken
+min = min floor
+f = first floor where egg breaks
+
+# call egg_drop with (arr, 0, 0)
+
+## egg_drop (arr, c, min)
+base case if n = 2
+# drop an egg from the ground floor
+    if it breaks (arr[0] == True)
```

Comment:
Loved the fact that you went above and beyond and wrote python code with the pseudocode for binary search.